sdk/eventgrid/azure-eventgrid/azure/eventgrid/_consumer.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from typing import TYPE_CHECKING
+
+from azure.core import PipelineClient
+from azure.servicebus import ReceivedMessage
```

Comment:
Do NOT import from servicebus publicly, since this would tight the usage of this SDK to servicebus installed

Reply:
will wait to fix until we discuss how to handle event handler message input, after I create samples for all message types.